### PR TITLE
Implement plot directive for DispersionPlot

### PR DESCRIPTION
**Pull request by lwgray**
Addresses #687 

This was a Simple implementation - just had to use the new ``load_hobbies`` function

### CODE REVIEW

File: docs/api/text/dispersion.rst [hunk 1]
```diff
@@ -5,16 +5,13 @@ Dispersion Plot
...
     from yellowbrick.text import DispersionPlot
-
-After importing the visualizer, we can :doc:`load the corpus <corpus>`
-
-.. code:: python
+    from yellowbrick.datasets import load_hobbies
 
     # Load the text data
-    corpus = load_corpus("hobbies")
+    corpus = load_hobbies()
```
Comment (lwgray) [inline docs/api/text/dispersion.rst]: Great job Ben, This makes it so much easier.

**Comment by bbengfort** [comment]
@lwgray not sure why Travis errored - I restarted the job.

**Comment by lwgray** [comment]
it's because the gallery needs that image file... i deleted it

**Comment by lwgray** [comment]
@rebeccabilbro Thanks for the heads up...

**Comment by bbengfort** [comment]
Ah that's right ... the gallery is going to be an issue.

**Pull request merged**